grafana-plugin/src/containers/UserSettings/UserSettings.tsx

```diff
@@ -48,18 +47,12 @@ const UserSettings = observer((props: UserFormProps) => {
     }
   }, [isDesktopOrLaptop]);
 
-  const getTabClickHandler = useCallback((tab: UserSettingsTab) => {
-    return () => {
-      onTabChange(tab);
-    };
-  }, []);
-
   const onTabChange = useCallback((tab: UserSettingsTab) => {
     setActiveTab(tab);
   }, []);
 
   const isModalWide =
-    (activeTab === UserSettingsTab.UserInfo && isDesktopOrLaptop) || activeTab === UserSettingsTab.PhoneVerification;
+    !isDesktopOrLaptop || activeTab === UserSettingsTab.UserInfo || activeTab === UserSettingsTab.PhoneVerification;
```

Comment:
Could you explain why `(activeTab === UserSettingsTab.UserInfo && isDesktopOrLaptop)` turned in `!isDesktopOrLaptop || activeTab === UserSettingsTab.UserInfo`  please

Reply:
After discussing with @raphael-batte we've decided to make the modal go full width for mobile devices and keep the logic for desktop as it was previously.

Reply:
got it, ok